serverless/pages/ml-nlp-auto-scale.mdx

```diff
@@ -0,0 +1,128 @@
+When you optimize for ingest, the throughput will be higher, which increases the number of inference requests that can be performed in parallel.
+When you optimize for search, the latency will be lower during search processes.
+
+- If you want to optimize for ingest, set the number of threads to `1` (`"threads_per_allocation": 1`).
+- If you want to optimize for search, set the number of threads to greater than `1`.
+Increasing the number of threads will make the search processes more performant.
+
+## Enabling autoscaling in Kibana - adaptive resources
+
+You can enable adaptive resources for your models when starting or updating the model deployment.
+Adaptive resources make it possible for Elasticsearch to scale up or down the available resources based on the load on the process.
+This can help you to manage performance and cost more easily.
+When adaptive resources are enabled, the number of vCUs that the model deployment uses is set automatically based on the current load.
+When the load is high, the number of vCUs that the process can use is automatically increased.
+When the load is low, the number of vCUs that the process can use is automatically decreased.
+
+You can choose from three levels of resource usage for your trained model deployment; autoscaling will occur within the selected level's range.
+
+Refer to the tables in the auto-scaling-matrix section to find out the setings for the level you selected.
+
+<DocImage size="xxl" url="../images/ml-nlp-deployment.png" alt="ML model deployment with adaptive resources enabled." />
+
+Search projects are given access to more processing resources, while Security and Observability projects have lower limits. This difference is reflected in the UI configuration: Search projects have higher resource limits compared to Security and Observability projects to accommodate their more complex operations.
+
+On Serverless, adaptive allocations are automatically enabled for all project types.
+However, the "Adaptive resources" control is not displayed in Kibana for Observability and Security projects.
+
+## Model deployment resource matrix
+
+The used resources for trained model deployments depend on three factors:
+
+- your cluster environment (Serverless, Cloud, or on-premises)
+- the use case you optimize the model deployment for (ingest or search)
+- whether model autoscaling is enabled with adaptive allocations/resources to have dynamic resources, or disabled for static resources
+
+The following tables show you the number of allocations, threads, and vCUs available on Serverless when adaptive resources are enabled or disabled.
+
+### Deployments on serverless optimized for ingest
+
+In case of ingest-optimized deployments, we maximize the number of model allocations.
+
+#### Adaptive resources enabled
+
+| Level  | Allocations                                          | Threads | vCUs                                                |
```

Comment:
The table is incorrect, because is Serverless we refer to VCUs, not VCPUs. 
Formulas: 
```
    // general purpose (c6gd) 1VCU = 1GB RAM / 0.5 vCPU
    // vector optimized (r6gd) 1VCU = 1GB RAM / 0.125 vCPU
```

Unfortunately it's not possible to know in Kibana what purpose serverless project is (general vs vector optimized), so we assume it's vector optimized and show VCUs number accordingly. You can see on the screenshot you used that for Low consumptions we use 16 VCUs.

Reply:
@arisonl should we refer to VCUs or VCPUs in the matrix for serverless?

Reply:
++ since we have the mapping to VCUs, let's do this conversion, as Dima indicates VCUs are the chargeable units in serverless.